marimo/_save/save.py

```diff
@@ -115,21 +114,13 @@ def _set_context(self, fn: Callable[..., Any]) -> None:
         # checking a single frame- should be good enough.
         f_locals = inspect.stack()[2 + self._frame_offset][0].f_locals
         self.scope = {**ctx.globals, **f_locals}
-        # In case scope shadows variables
-        #
-        # TODO(akshayka, dmadisetti): rewrite function args with an AST pass
-        # to make them unique, deterministically based on function body; this
-        # will allow for lifting the error when a ShadowedRef is also used
-        # as a regular ref.
-        for arg in self._args:
-            self.scope[arg] = ShadowedRef()
 
         # Scoped refs are references particular to this block, that may not be
         # defined out of the context of the block, or the cell.
         # For instance, the args of the invoked function are restricted to the
         # block.
         cell_id = ctx.cell_id or ctx.execution_context.cell_id or ""
-        self.scoped_refs = set(self._args)
+        self.scoped_refs = set([f"*{k}" for k in self._args])
```

Comment:
Can you store the prefix `*` used in this file and elsewhere in a constant, and refer to the constant instead?